openupgrade_scripts/scripts/base/15.0.1.3/pre-migration.py

```diff
@@ -34,4 +34,40 @@ def migrate(cr, version):
             "recommended to run the Odoo with --load=openupgrade_framework "
             "when migrating your database."
         )
-        openupgrade.rename_xmlids(cr, [("mail.icp_mail_catchall_alias", "base.icp_mail_catchall_alias")])
+
+    openupgrade.rename_xmlids(
+        cr,
+        [
+            # From l10n_ec
+            ("l10n_ec.state_ec_1", "base.state_ec_01"),
```

Comment:
nitpicking (non blocking) : usually, in Openupgrade, we define a variable at the top of the file, and then just call ``rename_xml_ids`` in the main function. It is so more readable to see all the steps of the main file.

see for exemple : https://github.com/OCA/OpenUpgrade/blob/14.0/openupgrade_scripts/scripts/base/14.0.1.3/pre-migrate.py#L23